packages/shared/src/index.ts

```diff
@@ -20,7 +20,6 @@ export * from './deprecated';
 export { deriveState } from './deriveState';
 export * from './error';
 export * from './file';
-export { handleValueOrFn } from './utils/handleValueOrFn';
```

Comment:
I guess this should be fine with https://github.com/clerk/javascript/pull/4972 merged now 👍

Reply:
This is a duped export

![Screenshot 2025-01-23 at 9 17 27 AM](https://github.com/user-attachments/assets/50e288c6-95fe-4f4b-b1e3-6c5413bde509)